ghc92/Prelude/Linear/Generically/Types.hs

```diff
@@ -0,0 +1,26 @@
+-- | Prior to GHC 9.4, linear-base defined its own versions of @Generically@ and
+-- @Generically1@. As a temporary workaround to enable compilation on both GHC
+-- 9.4 and 9.2, this module exposes linear-base's own implementations of those
+-- types, while the 9.4 version simply re-exports @Generics.Linear@.
+module Prelude.Linear.Generically.Types
```

Comment:
I suppose this module (and its alter-ego in `ghc94/`) should be an `Internal` module, shoudn't it?